jupyter_scheduler/tests/test_execution_manager.py

```diff
@@ -1,50 +1,55 @@
-import os
-from contextlib import contextmanager
 from pathlib import Path
-from unittest.mock import PropertyMock, patch
 
 import pytest
 from sqlalchemy import create_engine
 from sqlalchemy.orm import sessionmaker
 
+from conftest import DB_URL
 from jupyter_scheduler.executors import DefaultExecutionManager
-from jupyter_scheduler.orm import Base, Job
+from jupyter_scheduler.orm import Job
 
-NOTEBOOK_DIR = Path(__file__).resolve().parent / "test_staging_dir" / "job-3"
+JOB_ID = "69856f4e-ce94-45fd-8f60-3a587457fce7"
 NOTEBOOK_NAME = "side_effects.ipynb"
+SIDE_EFECT_FILE_NAME = "output_side_effect.txt"
+
+NOTEBOOK_DIR = Path(__file__).resolve().parent / "test_staging_dir" / "job-4"
```

Comment:
We should have a fixture for the staging directory which uses the default staging directory name in Jupyter Scheduler:

```py
@pytest.fixture
def staging_dir(jp_data_dir):
    return os.path.join(jp_data_dir, "scheduler_staging_area")
```

For the side effect test:

1. Move `output_side_effect.txt` and `side_effects.ipynb` under a new directory containing static files used in tests, e.g. `jupyter_scheduler/tests/static`.
2. Write a new `staging_dir_with_side_effects` fixture that copies these files to `staging_dir`, and returns a two-tuple of the input URI and the side effect file. So this fixture should return `('side_effects.ipynb', 'output_side_effect.txt')`